code/modules/ruins/lavalandruin_code/ash_walker_den.dm

```diff
@@ -76,6 +76,7 @@
 
 /obj/effect/mob_spawn/human/ash_walker/special(mob/living/carbon/human/new_spawn)
 	new_spawn.rename_character(new_spawn.real_name, new_spawn.dna.species.get_random_name(new_spawn.gender))
+	new_spawn.mind.offstation_role = TRUE
```

Comment:
Much better, thanks for changin in

Reply:
Thank you for reviewing it and noticing my mistake :)